WalletWasabi.Tests/UnitTests/BitcoinCore/RpcBasedTests.cs

```diff
@@ -15,7 +15,7 @@ public class RpcBasedTests
 	{
 		#region Mocked RPC response
 
-		public const string RpcOutput = @"
+		public static string RpcOutput = @"
```

Comment:
Does this need to be static? I tried with readonly and it passes, but
I got a crazy Naming Rule Violation and IntelliSense wants me to change its name to ____RULE_VIOLATION____RPCOUTPUT____RULE_VIOLATION____, i don't know what happened there, but i'm pretty sure we don't name our readonly vars like this

Reply:
> i'm pretty sure we don't name our readonly vars like this

That's correct.